plugins/Ecommerce/Controller.php

```diff
@@ -77,7 +77,7 @@ public function getConversionsOverview()
         $view = new View('@Ecommerce/conversionOverview');
         $idGoal = Common::getRequestVar('idGoal', null, 'string');
 
-        $goalMetrics = Request::processRequest('Goals.get', array('idGoal' => $idGoal));
+        $goalMetrics = Request::processRequest('Goals.get', array('idGoal' => $idGoal, 'filter_limit' => '-1'), $default = []);
```

Comment:
Actually, I wonder if we also need to set always `filter_offset` in case it is sent along  the request somehow?

Reply:
I think by setting `default=[]` likely not needed, forgot about it.

Reply:
`$default = []` changes the default values of the request from being `$_GET + $_POST` to `[]`, so if filter_limit/filter_offset is sent in the request, it will be ignored. I'm pretty sure filter_limit = -1 isn't necessary w/ default, but like was discussed before, can't hurt.